src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -820,4 +817,10 @@
   <data name="NewTabMenuFolderEmpty" xml:space="preserve">
     <value>Empty...</value>
   </data>
-</root>
+  <data name="ClosePaneText" xml:space="preserve">
+    <value>Close Pane</value>
+  </data>
+  <data name="ClosePaneToolTip" xml:space="preserve">
+    <value>Close the active pane if multiple panes are present</value>
+  </data>

```

Comment:
Do we need a tooltip here? Personally, I think this tooltip doesn't add much value. Like, we should change it to "Close the active pane" because it works regardless of how many panes are present, but at that point, we have a "close pane" menu item with supplemental text of "close the active pane".

Reply:
People have been salty about the lack of tooltips before (#13243), so I'm gonna double down that the tooltips _are_ important

Reply:
I think that the tooltip doesn't hurt, so I'd opt to keep it in.

Reply:
I think if we opt to keep the functionality only possible for when multiple panes are open, then it makes sense to leave the tool tip as is with the additional context of 'if multiple panes are present'.